test-support/prod-simulator-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
     ports:
       - 8888:9000
     environment:
-      - SECRET_KEY=notarealsecret
+      - SECRET_KEY=K5SgucxBYC3xJwNcGWZV1Y7uASmrc+SqftbcTROH0s8=
```

Comment:
**nitpick:** Maybe add a comment above that this is a randomly generated secret that isn't real but complies with play's secret length